### ISSUE TITLE
Empty string in boolean attributes and values is treated as None in tupleparser

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1032]
In values, NULL is represented as omitting the VALUE element, so having a second way of representing that is redundant and is also not defined in DSP0201.

`None` in attributes means they are unspecified and possibly default to something. Specifying an attribute with an empty value to mean the same thing is redundant and is also not defined in DSP0201.

**Discussion point:** Is there a need for supporting the empty string?

**Comment by andy-maier** [comment]
We agreed that we did not see a need for the empty string.
Also, I verified that the SAX parser rejects `ATTR` and `ATTR=` attempts of trying to get to an empty attribute value.

So the only remaining possibility is that a WBEM server actually specifies `ATTR=""` for a boolean attribute. That would violate CIM-XML, but it cannot be excluded that such servers exist.

I think deprecating the support for it would be more convenient than removing the support altogether, just in case somebody has used it.
